fairlearn/metrics_plots/plot_disparities_in_metric.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+from fairlearn.metrics import MetricFrame
+
+
+def plot_disparities_in_metric(metric, y_true, y_pred, sensitive_features, show_plot=True):
```

Comment:
The plot functions should probably accept and return an `ax` parameter to receive and return the `matplotlib`'s `axis` object so that users can further customize the plots.

Reply:
Sounds good. I compared with some of the functions in `scikit-learn`. They don't have a `show_plot` arg, but rather the caller is expected to call `plt.show()` themselves. Is that preferable? I find it kind of odd to call a function called `plot_*` and then there's no plot shown unless I do something more, but perhaps that's just me.

Reply:
In the context of matplotlib, I find it nice if the caller has the capability of having each plot in a subplot (hence passing `ax`), and makes it more configurable when the method returns an object which the user can change. In terms of the user having to call `plt.show()`, I'm agnostic. I don't mind having the `show_plot` as you have here. But it's basically replacing a single line of code. But I also don't mind having it `True` by default, which is kinda nice to the user I guess.

Reply:
I'll leave it in there for now then, unless someone has stronger feelings about this :-)